src/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration/OpenApiDocumentGenerationSettings.cs

```diff
@@ -17,11 +17,43 @@ public class OpenApiDocumentGenerationSettings
         /// </summary>
         /// <param name="schemaGenerationSettings">The schema generation settings.</param>
         public OpenApiDocumentGenerationSettings(SchemaGenerationSettings schemaGenerationSettings)
+            : this(schemaGenerationSettings, false)
+        {
+        }
+
+        /// <summary>
```

Comment:
Should we have this controller? This will allow people to not give us schemaGenerationSettings.

Reply:
which is fine bc if people want to use default resolver but only want to set the bool this would work. if they want to set both there is another overload available

---
In reply to: [253721269](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations/pull/198#discussion_r253721269) [](ancestors = 253721269)